### PR TITLE
Updates to pt-br language files

### PR DESCRIPTION
**Pull request by scr0llbaer9000**
Updates to the pt-br language files, as submitted by João Coelho on Discord.

### CODE REVIEW

**Comment by nittka** [comment]
I cannot check the translations themselves. I created a PR with minor fixes to the spacing, making the structure consistent.

File: res/lang/news/news_pt-br.txt [hunk 1]
```diff
@@ -60,30 +60,30 @@ BOMB_DETONATION_IN_TVTOWER_FOUND_CLUES_TEXT  = Autoridades investigaram o atenta
...
 NEWS_ANNOUNCE_MOVIE_ACTOR_IS_DIRECTOR_DESCRIPTION = Diretor e ator principal %DIRECTORNAME1% tem um novo filme «%MOVIETITLE%» na produção.
 
-#NEWS_ANNOUNCE_MOVIE_NO_CAST_TITLE       = 
-#NEWS_ANNOUNCE_MOVIE_NO_CAST_DESCRIPTION = 
+NEWS_ANNOUNCE_MOVIE_NO_CAST_TITLE       = Em breve no cinema: «%MOVIETITLE%»
+NEWS_ANNOUNCE_MOVIE_NO_CAST_DESCRIPTION = Publicado o primeiro trailer de «%MOVIETITLE%».
 
-#NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_TITLE       = 
-#NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_DESCRIPTION = 
+NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_TITLE       = Em breve no cinema: «%MOVIETITLE%»
```
Comment (GWRon) [inline res/lang/news/news_pt-br.txt:66]: the quotation marks differ to the one in 
`TOASTMESSAGE_BOMB_LICENCE_DESTROYED_TEXT     = A explosão destruiu '%TITLE%'.`

File: res/lang/news/news_pt-br.txt [hunk 2]
```diff
@@ -60,30 +60,30 @@ BOMB_DETONATION_IN_TVTOWER_FOUND_CLUES_TEXT  = Autoridades investigaram o atenta
...
+NEWS_ANNOUNCE_MOVIE_NO_DIRECTOR_DESCRIPTION = O ator principal %ACTORNAME1% em breve será apresentado nos cinemas com «%MOVIETITLE%».
```
Comment (GWRon) [inline res/lang/news/news_pt-br.txt:67]: quotation marks.

File: res/lang/news/news_pt-br.txt [hunk 3]
```diff
@@ -60,30 +60,30 @@ BOMB_DETONATION_IN_TVTOWER_FOUND_CLUES_TEXT  = Autoridades investigaram o atenta
...
 
 NEWS_ANNOUNCE_MOVIE_TITLE1 = Novo filme com %ACTORNAME1%
 NEWS_ANNOUNCE_MOVIE_TITLE2 = %DIRECTORNAME1% dirige novo filme
-#NEWS_ANNOUNCE_MOVIE_TITLE3 = 
+NEWS_ANNOUNCE_MOVIE_TITLE3 = %DIRECTORNAME1% com novo filme nos cinemas
 
 NEWS_ANNOUNCE_MOVIE_DESCRIPTION1 = O diretor %DIRECTORNAME1% e o ator %ACTORNAME1% entregam o filme «%MOVIETITLE%» às telas.
 NEWS_ANNOUNCE_MOVIE_DESCRIPTION2 = Em breve nos cinemas: %ACTORNAME1% em «%MOVIETITLE%». Ator %ACTORNAME2% está todo orgulhoso com o filme.
 NEWS_ANNOUNCE_MOVIE_DESCRIPTION3 = Últimos acabamentos para o filme «%MOVIETITLE%». %ACTORNAME1% está orgulhoso de estrelar o filme.
 NEWS_ANNOUNCE_MOVIE_DESCRIPTION4 = Primeiros trailers de «%MOVIETITLE%» disponíveis. %DIRECTORNAME1%: "Melhor filme do ano".
```
Comment (GWRon) [inline res/lang/news/news_pt-br.txt:79]: here `" ... "` is used, so we have:
```
' ... '
" ... "
« ... »
```
in the mix already.

File: res/lang/bossbetty/bossbetty_pt-br.txt [hunk 4]
```diff
@@ -78,34 +78,34 @@ BETTY_PRESENT_9            = Anel
...
-#AWARDNAME_CUSTOMPRODUCTION   = 
-#AWARD_WON                    = 
-#YOU_WON_THE_AWARDNAME        = 
-#PLAYERNAME_WON_THE_AWARDNAME = 
-#NOBODY_WON_THE_AWARDNAME     = 
-#CHANNEL_IMAGE                = 
-#FOR_X_HOURS                  = 
-#FOR_X_HOUR                   = 
-#ATTRACTIVITY                 = 
+AWARDNAME_CULTURE            = Sammy "pela Cultura"
```
Comment (GWRon) [inline res/lang/bossbetty/bossbetty_pt-br.txt:81]: quotation marks - here it is `" ... "` - use the same across the localization please.

Comment (scr0llbaer9000) [reply res/lang/bossbetty/bossbetty_pt-br.txt:81]: he didn't change the quotation marks from the existing languages, so not his fault.

it's the same in all the other language files too.

we'd have to change and harmonize the quotation marks for every language then.

Comment (GWRon) [reply res/lang/bossbetty/bossbetty_pt-br.txt:81]: yes, other localisations might need harmonization too ...

BUT ... not all quotation marks are "in use" in other languages. So double quots are I guess non-English, the sidewards looking arrows are more a "Swiss thing" but also known in German.

At the end quotation marks should be "unified" on a "per language" base.

**Comment by scr0llbaer9000** [comment]
I'd say let's merge this PR now, and for the quotation marks let's make a new PR (as this one is referring to the pt-br localization only).

**Comment by nittka** [comment]
I'd also be in favour of merging the content and doing the quotes cleanup separately. @GWRon ?

**Pull request merged**